dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -772,12 +776,18 @@
         <Setter Property="FontSize" Value="{ThemeResource ControlContentThemeFontSize}" />
         <Setter Property="UseSystemFocusVisuals" Value="True" />
         <Setter Property="HorizontalContentAlignment" Value="Stretch" />
+        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="local:NavigationViewItem">
                     <Grid 
                         x:Name="LayoutRoot" 
                         Background="{TemplateBinding Background}"
+                        contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
+                        contract7NotPresent:CornerRadius="{ThemeResource OverlayCornerRadius}"
```

Comment:
In style Setter you use ControlCornerRadius but here (if rs5 and below) you're using OverlayCornerRadius. Shouldn't it be the same value for both?:)

Reply:
You're right! Fixed